tests/providers/aws/e2e.py

```diff
@@ -304,6 +310,71 @@ def tearDownClass(cls):
             str(exception))) from exception
       logger.info('Volume {0:s} successfully deleted.'.format(volume.volume_id))
 
+class S3EndToEndTest(unittest.TestCase):
+  """End to end test on AWS.
+  To run this tests, add your project information to a project_info.json file:
+  {
```

Comment:
Since this is now a separate class dedicated to S3, shouldn't `testCopyEBSSnapshotToS3` be moved here?

Reply:
I think once we rewrite the E2E tests, it would move to it's own separate class entirely.